docs/_user_guide/best_practices_monitor_usage.md

```diff
@@ -152,127 +137,42 @@ system.time({
-You should see:
-```bash
-Loading required package: iterators
-Loading required package: parallel
-   user  system elapsed
-297.135   1.781 123.176
-Parallel NPV Calculation (using 8 cores):
-       V1
- Min.   :-709.5883
- 1st Qu.: -96.3171
- Median :   0.2777
- Mean   :   0.1964
- 3rd Qu.:  96.8101
- Max.   : 754.5522
-```
-
-While the program is running, you should see 8 R processes running in the `htop` output because we
-specified 8 cores in our R program and about 8 CPU cores being utilized in `userload` output. The program will run faster since we are using 8 cores instead of 1 but does not get you 8X speedup because of parallelization overhead.
-
-![](/images/monitor-2.png)
-
-Last modification we are going to make is to pass the number of cores as a command line argument to our R script.
-Save the following to a new script called `investment-npv-parallel-args.R`.
-
-```R
-#!/usr/bin/env Rscript
-############################################
-# This script accepts a user specified argument to set the number of cores to run on
-# Run from the command line:
-#
-#      Rscript investment-npv-parallel-args.R 8
-#
-# this will execute on 8 cores
-###########################################
-# accept command line arguments and save them in a list called args
-args = commandArgs(trailingOnly=TRUE)
-library(foreach)
-library(doParallel)
-
-options(warn=-1)
-# set the number of cores here from the command line. Avoid using detectCores() function.
-ncore <- as.integer(args[1])
-
-# register parallel backend to limit threads to the value specified in ncore variable
-registerDoParallel(ncore)
-
-# define function for NPV calculation
-npv_calculation <- function(cashflows, discount_rate) {
-  # inputs: cashflows (a vector of cash flows over time) and discount_rate (the discount rate).
-  npv <- sum(cashflows / (1 + discount_rate)^(0:length(cashflows)))
-  return(npv)
-}
-
-# number of trials
-num_trials <- 500000
-
-# measure the execution time of the Monte Carlo simulation
-system.time({
-  # use the foreach package to loop through the specified number of trials (num_trials) in parallel
-  # within each parallel task, random values for input parameters (cash flows and discount rate) are generated for each trial
-  # these random input values represent different possible scenarios
-  results <- foreach(i = 1:num_trials, .combine = rbind) %dopar% {
-    # randomly generate input values for each trial
-    cashflows <- runif(10000, min = -100, max = 100)  # random cash flow vector over 10,000 time periods.
-    # these cash flows can represent costs (e.g., initial investment) and benefits (e.g., revenue or savings) associated with the project
-    discount_rate <- runif(1, min = 0.05, max = 0.15)  # random discount rate at which future cash flows are discounted
+!!! Notes
+    If you have not installed `foreach` and `doParallel` packages. You have to run these commands after loading `r`: `install.packages("foreach")` and `install.packages("doParallel")`
```

Comment:
If you have not installed `foreach` and `doParallel` packages for the version of R that you have loaded, you have to run these commands after loading `R` module: `install.packages("foreach")` and `install.packages("doParallel")`